.idea/deploymentTargetDropDown.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
all files from idea folder should be excluded from repo and added into gitignore

Reply:
done